apps/pie-docs/src/components/tag/overview/overview.md

```diff
@@ -32,7 +34,8 @@ Tags can be embedded in other components such as cards, data tables (among other
     dont: {
         type: usageTypes.text,
         items: [
-            "Don’t use when an interaction is required, use a [chip](/components/chip/) if needed."
+            "Don’t use for interactions crucial for the flow.",
+            "If only an icon is required to convey the message, use the tag: icon only component instead."
```

Comment:
If this makes sense to design then no issues - but I don't really understand the wording here. Is it a figma option?

Reply:
Currently, when we support this in engineering, it will link to another page.

from the ticket -> 

> Note: the don’ts section has a link to icon only documentation. This will be implemented in DSW-2550 so for now just ignore adding the link.